UnityProject/Assets/Scripts/UI/Systems/Lobby/CharacterCustomization.cs

```diff
@@ -1165,6 +1166,7 @@ public void OnManualNameChange()
 
 		private string TruncateName(string proposedName)
 		{
+			proposedName = System.Threading.Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase(proposedName.ToLower());
```

Comment:
I'm not sure we want this restriction. What about machine-like names when we have borgs and the like?

Reply:
Also the way you get to ``ToTitleCase`` looks convoluted. You could have just imported the ``System.Globalization`` namespace by adding ``Using System.Globalization`` to the top of the file.

Reply:
That's a fair point, I guess I was just going off of the original bug report and the fact that the text box doesn't let you enter multiple capital characters in a row.